composer/yamls/models/bert-base.yaml

```diff
@@ -12,20 +12,27 @@ train_dataset:
     use_masked_lm: true
     mlm_probability: 0.15
     max_seq_length: 128
-val_dataset:
-  lm:
-    split: validation
-    datadir:
-      - /datasets/wikipedia_saved_bert_128
-      - /datasets/bookcorpus_saved_bert_128
-    tokenizer_name: bert-base-uncased
-    seed: 19
-    shuffle: true
-    drop_last: false
-    subsample_ratio: 0.02
-    use_masked_lm: true
-    mlm_probability: 0.15
-    max_seq_length: 128
+evaluators:
+  - evaluator:
+      label: bert-base
+      eval_dataset:
+        lm:
+          split: validation
+          datadir:
+            - /datasets/wikipedia_saved_bert_128
+            - /datasets/bookcorpus_saved_bert_128
+          tokenizer_name: bert-base-uncased
+          seed: 19
+          shuffle: true
+          drop_last: false
+          subsample_ratio: 0.02
+          use_masked_lm: true
+          mlm_probability: 0.15
+          max_seq_length: 128
+      metric_names:
+        - Accuracy
+        - BinaryF1Score
+        - MatthewsCorrCoef
```

Comment:
BERT-base requires `LanguageCrossEntropy` and `MaskedAccuracy`, not any of these metrics